csp-mixin/signals/gcpce.libsonnet

```diff
@@ -0,0 +1,129 @@
+local commonlib = import 'common-lib/common/main.libsonnet';
+function(this)
+  {
+    local s = self,
+    filteringSelector: 'job!=""',
+    groupLabels: ['job', 'project_id'],
+    instanceLabels: ['instance_name'],
+    aggLevel: 'instance',
+    discoveryMetric: {
+      stackdriver: 'stackdriver_gce_instance_compute_googleapis_com_instance_cpu_utilization',
+    },
+    signals: {
+      // instancesText: {
+      //   name: "Instances",
+      //   type: 'text',
+      //   content: 'Use this section to look at one instance at a time by picking a value in the *Instance* picker at the top of the dashboard.'
+      //   // To Do: check if this works???
+      // },
```

Comment:
Is there any text panel type?

Reply:
'type' is for metric types here: gauge, counter, histogram, info, or raw.